app/src/views/Actors.vue

```diff
@@ -598,6 +628,36 @@ export default class ActorList extends mixins(DrawerMixin) {
       });
   }
 
+  async checkActorExist(name: string) {
+    const result = await ApolloClient.query({
+      query: gql`
+        query($query: ActorSearchQuery!, $seed: String) {
+          getActors(query: $query, seed: $seed) {
+            items {
+              ...ActorFragment
+            }
+            numItems
+          }
+        }
+        ${actorFragment}
+      `,
+      variables: {
+        query: {
+          query: name,
+          sortBy: "relevance",
+        },
+        seed: localStorage.getItem("pm_seed") || "default",
+      },
+    });
+
+    // Verify that the highest relevance ES result is an exact name match
+    if (result.data.getActors.items.length > 0 && result.data.getActors.items[0].name == name) {
```

Comment:
Triple equals for consistency

```suggestion
    if (result.data.getActors.items.length > 0 && result.data.getActors.items[0].name === name) {
```